src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -206,9 +208,46 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
   uint64_t end_ns = thread_state_slice->end_timestamp_ns();
   tooltip += absl::StrFormat(
       "<br/>"
-      "<b>Time:</b> %s",
+      "<b>Time:</b> %s"
+      "<br/><br/>",
       orbit_display_formats::GetDisplayTime(TicksToDuration(begin_ns, end_ns)));
 
+  if (thread_state_slice->switch_out_or_wakeup_callstack_id().has_value()) {
```

Comment:
Consider returning early if the callstack_id doesn't have value, seems easier to read.